package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "@types/node": "^20.4.4"
+  }
+}
```

Comment:
Is this file necessary? I was able to build and run without it. Not opposed just wondering what it's for in the root

Reply:
I have deleted this file, it was a leftover artifact from testing locally, thank you for catching it!